src/engraving/libmscore/spanner.cpp

```diff
@@ -757,6 +757,14 @@ void Spanner::computeEndElement()
 
     case Anchor::CHORD:
     case Anchor::NOTE:
+        if (!_endElement) {
+            ChordRest* cr = score()->findCR(tick2(), track2() / VOICES);
+            if (cr && _anchor == Anchor::NOTE && cr->isChord()) {
+                _endElement = toChord(cr)->upNote();
```

Comment:
when pasting a note-anchored line, find the "most" likely candidate for attaching the other end to, though this could definitely be improved